qt/applications/workbench/mantidworkbench.cpp

```diff
@@ -111,6 +123,9 @@ decltype(boost::this_process::environment()) childEnvironment(const fs::path &di
 #if defined(QT_PLUGIN_PATH)
   env["QT_PLUGIN_PATH"] = absolutePath(QT_PLUGIN_PATH, dirOfExe);
 #endif
+
+#endif
```

Comment:
Can we comment the `endif` names when there are multiple in succession:
```c++
#endif // QT_PLUGIN_PATH
#endif // NOT_CONDA
```

Reply:
:+1: Sounds good